qa/rpc-tests/sprout_sapling_migration.py

```diff
@@ -101,7 +99,7 @@ def run_migration_test(self, node, sproutAddr, saplingAddr, target_height):
         assert_equal(target_height, result['target_height'])
         assert_equal(1, result['result']['num_tx_created'])
         assert_equal(1, len(result['result']['migration_txids']))
-        assert_true(result['result']['amount_migrated'] > Decimal('0'))
+        assert_true(Decimal(result['result']['amount_migrated']) > Decimal('0'))
```

Comment:
@daira What are your thoughts on this line? I was a bit concerned casting two params in an assert_true.

Reply:
Looks ok to me.